PLUGINS.md

```diff
@@ -156,4 +257,48 @@ The produced result xml will be:
 </manifest>
 ```
 
+### Manual Steps After Installation
```

Comment:
I have skipped this comment. :) IMHO, it's best to make people check and fix their files before anything goes wrong.

In theory, they should not be in a situation where anything goes wrong but, alas, people are very inventive :)